codalab/apps/api/views/worksheet_views.py

```diff
@@ -6,42 +6,19 @@
 import traceback
 import mimetypes
 
-from uuid import uuid4
-
-from rest_framework import (permissions, status, viewsets, views, generics, filters)
-from rest_framework.decorators import action, link, permission_classes
-from rest_framework.exceptions import PermissionDenied, ParseError
+from rest_framework import (status, views)
 from rest_framework.response import Response
 
 from django.conf import settings
-from django.contrib.auth.decorators import login_required
-from django.contrib.contenttypes.models import ContentType
-from django.contrib.sites.models import Site
-from django.core.exceptions import ObjectDoesNotExist
-from django.core.exceptions import PermissionDenied as DjangoPermissionDenied
-from django.core.files.base import ContentFile
-from django.core.mail import EmailMultiAlternatives
-from django.core.mail import send_mail
-from django.http import Http404, StreamingHttpResponse
-from django.template import Context
-from django.template.loader import render_to_string
-from django.utils.decorators import method_decorator
```

Comment:
Even though we are not using some of these inports bring back some of the more useful ones.  `PermissionDenied, ContentFile, EmailMultiAlternatives, send_mail, http imports` etc..